src/components/RadioButton/RadioButton.spec.jsx

```diff
@@ -0,0 +1,112 @@
+import _ from 'lodash';
+import assert from 'assert';
+import React from 'react';
+import SyntheticEvent from 'react/lib/SyntheticEvent';
+import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+
+import describeWithDOM from '../../util/describe-with-dom';
+import { common } from '../../util/generic-tests';
+
+import RadioButton from './RadioButton';
+
+describe('RadioButton', () => {
+	common(RadioButton);
+
+	const booleanValues = [true, false];
+
+	describe('props', () => {
+		describe('isDisabled', () => {
+			it('sets the `disabled` attribute of the native radio button element.', () => {
+				_.forEach(booleanValues, (testValue) => {
+					const wrapper = shallow(<RadioButton isDisabled={testValue} />);
+
+					assert.equal(wrapper.find('input[type="radio"]').prop('disabled'), testValue);
+				});
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('isDisabled'), false);
+			});
+		});
+
+		describe('isSelected', () => {
+			it('sets the `checked` attribute of the native radio button element.', () => {
+				_.forEach(booleanValues, (testValue) => {
+					const wrapper = shallow(<RadioButton isSelected={testValue} />);
+
+					assert.equal(wrapper.find('input[type="radio"]').prop('checked'), testValue);
+				});
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('isSelected'), false);
+			});
+		});
+
+		describe('onSelect', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('onSelect'), _.noop);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props not defined in `propTypes` to the native input.', () => {
```

Comment:
I had thought about it: we would need to be able to pass in the names of the component's actual props so we know what _shouldn't_ be passed through and we would also need to be able to pass through the selector somehow since in some cases the native is the root of the render whereas in others it's a child of the root (like here).

One point of trouble might be if for some reason on some components we need to pass through some of those defined props down to the native. We haven't done enough of these to know if it's an edge case or not.